ocs_ci/helpers/storage_agnostic_dr_helpers.py

```
@@ -0,0 +1,866 @@
+
+def create_psk_secret(namespace, psk_value, cluster_config=None):
+    """
+    Create the VolSync PSK secret in a namespace on a cluster.
+
+    Args:
+        namespace (str): Namespace to create the secret in.
+        psk_value (str): Base64-encoded PSK value to use.
+        cluster_config: Cluster config object to run commands against.
+            Defaults to the current context if None.
+    """
+    run_cmd(
```

Ruff (0.15.17)

[error] Function call with `shell=True` parameter identified, security issue

(S604)